src/components/flowchart/draw.js

```diff
@@ -121,11 +121,10 @@ export const drawNodes = function (changed) {
     nodeActive,
     nodeSelected,
     hoveredParameters,
-    nodesWithInputParams,
+    nodeParameters,
```

Comment:
What is the cause behind this name change? The old name, `nodesWithInputParams`, is more intuitive than `nodeParameters`.